test/src/unit-conversions.cpp

```diff
@@ -189,48 +189,6 @@ TEST_CASE("value conversion")
         }
     }
 
-    SECTION("get an object (implicit)")
```

Comment:
Maybe we should add more `#ifdef` blocks rather than reordering the file?

Reply:
What is the benefit?

Reply:
The benefit is to maintain the order of the tests.